tests/system/conftest.py

```diff
@@ -120,35 +138,20 @@ def bq_client(credentials, use_mtls):
     return bigquery.Client(credentials=credentials)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def cleanup_datasets(bq_client: bigquery.Client):
+    for dataset in bq_client.list_datasets():
+        if prefixer.should_cleanup(dataset.dataset_id):
+            bq_client.delete_dataset(dataset, delete_contents=True, not_found_ok=True)
```

Comment:
This is pre-scrub, correct?

Reply:
That's right. Not 100% necessary, but I did notice we occasionally get some stray datasets that don't get post-scrubbed.